### DISCUSSION TITLE
remove dashboard

### DISCUSSION THREAD

**Discussion by ribbbo** [filamentphp/filament#12373] [Help]
### Package

Panel builder

### Package Version

v2

### How can we help you?

how to remove the default dashboard.
if I commented this Pages\Dashboard::class, on my config/filament.php I got 404 not found error.
![image](https://github.com/filamentphp/filament/assets/136681505/065758b3-63e6-47fd-911a-065bd27bcd47)

**Comment by leandrocfe** [comment]
https://filamentphp.com/docs/2.x/admin/dashboard/getting-started#customizing-the-dashboard-page

**Reply by ribbbo** [reply]
It's still has a Dashboard in my Admin panel

**Reply by ribbbo** [reply]
what I want is to remove in my Navigation the Dashboard Section

**Reply by leandrocfe** [reply]
ahh ok. You can remove it and change the `'home_url'` param in the config/filament.php file
